ckanext/dhis2harvester/dhis2_api.py

```diff
@@ -26,6 +25,13 @@
                                      'displayProperty=NAME&'
                                      'hierarchyMeta=true&'
                                      'outputIdScheme=UID',
+        "PIVOT_TABLES_CSV_INDICATOR_RESOURCE": 'analytics.csv?'
+                                               'dimension=dx:{indicators}&'
+                                               'dimension=pe:{periods}&'
+                                               'dimension=ou:{organisation_units}&'
+                                               'displayProperty=NAME&'
+                                               'hierarchyMeta=true&'
+                                               'outputIdScheme=UID',
```

Comment:
Still think duplication here is not a good idea :wink:  readability problems can be solved in multiple ways, (better named variables/comments/etc...) but code duplication can't be. Anyway, again probably a discussion for another time.